code/modules/uplink/uplink_items.dm

```diff
@@ -2099,3 +2099,10 @@ datum/uplink_item/role_restricted/superior_honkrender
 	item = /obj/item/storage/fancy/cigarettes/cigpack_syndicate
 	cost = 2
 	illegal_tech = FALSE
+
+/datum/uplink_item/role_restricted/traitorous_cloner
+	name = "Defective Cloner"
+	desc = "A box that cointains boards and tools to setup defective clone machinery for making defective clones."
+	item = /obj/item/storage/box/syndicate/traitorous_cloner
+	cost = 4
```

Comment:
I think the cost should be upped. You will really only use it with emag anyway but 4 is too low for making your blood bros.

Reply:
You need emag to make blood bro, so cost is upped, other way you get stupid clones that basically do nothing but impersonating some random dude they are clones of.